authors/apps/articles/serializers.py

```diff
@@ -69,7 +69,8 @@ class Meta:
             'favorited',
             'favorite_count',
             'comments',
-            'tagList'
+            'tagList',
+            'author'
```

Comment:
@3Nakajugo We should only return the username here. Not all user details.

#### Screenshot 
<img width="748" alt="Screen Shot 2019-05-07 at 11 28 37" src="https://user-images.githubusercontent.com/36948493/57285886-d7e85180-70bc-11e9-8607-f5b97f6caca7.png">